vllm/distributed/parallel_state.py

```diff
@@ -342,6 +346,62 @@ def broadcast_object_list(self,
                                                 group=self.device_group)
         return obj_list
 
+    def send_object(self,
+                    obj: Any,
+                    dst: int,
+                    group: Optional[ProcessGroup] = None) -> None:
+        """Send the input object list to the destination rank."""
+        assert dst < self.world_size, f"Invalid dst rank ({dst})"
+
+        # Bypass the function if we are using only 1 GPU.
+        if self.world_size == 1:
```

Comment:
if world size is 1, can we send object? to whom?

Reply:
This was a holdover from broadcast, but doesn't make sense in this context. Removed.